packages/yarnpkg-core/sources/MessageName.ts

```diff
@@ -98,6 +98,9 @@ export enum MessageName {
   CONSTRAINTS_CHECK_FAILED = 84,
   UPDATED_RESOLUTION_RECORD = 85,
   EXPLAIN_PEER_DEPENDENCIES_CTA = 86,
+  MIGRATION_SUCCESS = 87,
```

Comment:
I kept the message name introduced in #5526 to avoid having the merge conflict to fix later on. I plan to land #5526 soon, I just want to wait enough for the benchmarks to clearly show the performance change (if I merged them now they'd be conflated with the metadata cache improvements).